internal/scaffold/ansible/go_mod.go

```diff
@@ -42,31 +42,40 @@ go 1.13
 
 require (
 	github.com/operator-framework/operator-sdk master
-	sigs.k8s.io/controller-runtime v0.3.0
+	sigs.k8s.io/controller-runtime master
```

Comment:
Should we use the master and not a tag?
Or we are here just waiting for the next tag release for we are able to move forward with this one?

Reply:
The latter. The k8s 1.16 bump was on controller-runtime `master`, but not yet released. I was waiting on the `v0.4.0` release, which is now out 🎉